_posts/2019-02-28-newpipe-0.16.0-released.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: NewPipe 0.16.0 released, adding comment support and mediaCCC service
+short: "NewPipe 0.16.0 released"
+date: 2019-02-28 19:00:00 +0200
+categories: [release, pinned]
+author: TobiGr
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We are happy to release NewPipe 0.16.0 bringing support for reading top-level YouTube comments. Moreover, with [media.ccc.de](https://media.ccc.de) NewPipe supports three services now!
+
+
+<!-- more -->
+
+
+## Comment support
+[@yausername](https://github.com/yausername) implemented one of the most requested features: support for reading comments!
+
+From now on, NewPipe can display comments underneath the video description. At the moment comment support is limited to reading top-level comments. But [@yausername](https://github.com/yausername) plans to extend it when he has some spare time left.
+
+Nevertheless, NewPipe can and will only support reading comments, because any other activity like writing or up-/down-voting comments require a YouTube account. For more on this topic, take a look at our [FAQ](/FAQ/bugs/).
+
+As always, NewPipe tries to be as much customizable as possible. In case do not want to read comments at all, feel free to disable them in the content settings.
+
+<img class="no-flow vertical" src="/img/screenshots/shot_video_detail_with_coments_tablet_dark.png"/>
+
+Along with adding the ability to view comments, [@yausername](https://github.com/yausername) added an autoplay toggle to the video pages. This allows you to easily turn on and off autoplay videos without going to the settings. You can find the toggle in the "related videos" tab by swiping the comment to the left.
+
+
+## media.ccc.de service
+
+[@theScrabi](https://github.com/theScrabi) completed the work he put into adding [media.ccc.de](https://media.ccc.de) before C35C. As a result, you are now able to listen and watch to all the interesting conference records. To do so, open the navigation menu, change the service to `mediaCCC`. You can now either choose from a long list of conferences or just use the search. 
+
+For those of you, who might not know the [Chaos Computer Club](https://www.ccc.de/en/), feel free take a look at their [Wikipedia article](https://en.wikipedia.org/wiki/Chaos_Computer_Club).
+
+<img class="no-flow" src="/img/screenshots/shot_navigation_drawer_services_dark.png"/>
+
+In case, you miss a service and want to add it to NewPipe or you are just interested how this can be done, please take a look at our documentation. [@theScrabi](https://github.com/theScrabi) has been constantly working on it the last few months. The docs are far from being perfect, so any feedback or help is welcome. At this point we'd like to thank [@snappyapple632](https://github.com/snappyapple632) for cleaning up and improving the docs slightly.
```

Comment:
No comma after "case". The second "you" after "NewPipe or" isn't really necessary and should be omitted.

```suggestion
In case you miss a service and want to add it to NewPipe or are just interested how this can be done, please take a look at our documentation. [@theScrabi](https://github.com/theScrabi) has been constantly working on it the last few months. The docs are far from being perfect, so any feedback or help is welcome. At this point we'd like to thank [@snappyapple632](https://github.com/snappyapple632) for cleaning up and improving the docs slightly.
```